### PR TITLE
DEVX-3237 New TTS language and style parameters

### PR DESCRIPTION
**Pull request by vshisterov**
# Description

New talk operation parameters added: `language` and `style`

# Checklist

- [x] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/voice.yml [hunk 1]
```diff
@@ -912,107 +914,62 @@ components:
...
+        - "th-TH"
+        - "tr-TR"
+        - "uk-UA"
+        - "vi-VN"
+        - "yue-CN"
+    style:
+      description: The voice style to use
+      type: integer
+      default: 0
+
```
Comment (bencgreenberg) [inline definitions/voice.yml:972]: I think we also want to include an `enum` here of the possible values for `style`

Comment (vshisterov) [reply definitions/voice.yml:972]: It's not an enumeration, it's a plain integer.

Comment (bencgreenberg) [reply definitions/voice.yml:972]: what i meant, is we should include a list of all the possible value

Comment (vshisterov) [reply definitions/voice.yml:972]: It depends on the selected `language`. For integer parameters, OpenAPI doesn't suggest to have list of values, though we may add min and max, still the max. which is 11 currently, will be relevant for only one language (`en-US`).

Comment (bencgreenberg) [reply definitions/voice.yml:972]: Ah, okay, given its variability and inconsistency, then I see the point. Thanks, Victor.

**Comment by bencgreenberg** [comment]
I'd like us to figure out what we are doing with `style` in the NDP PR before moving forward here.

File: definitions/voice.yml [hunk 2]
```diff
@@ -912,107 +914,62 @@ components:
...
+        - "ru-RU"
+        - "sk-SK"
+        - "sv-SE"
+        - "th-TH"
+        - "tr-TR"
+        - "uk-UA"
+        - "vi-VN"
+        - "yue-CN"
+    style:
+      description: The voice style to use
```
Comment (bencgreenberg) [inline definitions/voice.yml:969]:
```suggestion
      description: The vocal style (vocal range, tessitura, and timbre) to use
```

**Pull request merged**